docs/accounting/shell.nix

```diff
@@ -0,0 +1,30 @@
+with (import <nixpkgs> {});
+
+let pkgs = import ../../pkgs.nix; in
+stdenv.mkDerivation {
```

Comment:
If you wanted to actually have a `default.nix` so you can `nix build` the whole thing you could add a build step. See e..g https://github.com/input-output-hk/plutus/blob/master/plutus-core-spec/default.nix

Reply:
Great! Do you pin your Nix packages set?

Reply:
Yes, but in an indirect way via a bunch of stuff at the root of the repo which I wouldn't worry about atm.